nullaway/src/test/java/com/uber/nullaway/NullAwayVarargsTests.java

```diff
@@ -0,0 +1,100 @@
+package com.uber.nullaway;
+
+import java.util.Arrays;
+import org.junit.Test;
+
+public class NullAwayVarargsTests extends NullAwayTestsBase {
```

Comment:
Possibly the same issue as the try-with-resources tests, but in this case I am happy to keep these tests separate, as the way we handle var args might change once we support array and generic nullability (and try to conform with JSpecify there)